app/src/main/java/org/wikipedia/settings/Prefs.kt

```diff
@@ -734,4 +734,8 @@ object Prefs {
     var recommendedContentSurveyShown
         get() = PrefsIoUtil.getBoolean(R.string.preference_key_recommended_content_survey_shown, false)
         set(value) = PrefsIoUtil.setBoolean(R.string.preference_key_recommended_content_survey_shown, value)
+
+    var hasUserVisitedDonorHistory
+        get() = PrefsIoUtil.getBoolean(R.string.preference_key_user_visited_donor_history_screen, false)
+        set(value) = PrefsIoUtil.setBoolean(R.string.preference_key_user_visited_donor_history_screen, value)
```

Comment:
I know we've discussed this during the meeting, but according to the [Figma design](https://www.figma.com/design/qZvcD1IoEjPP13X5Ed3Fmv/Android-%E2%86%92-Contributions-Dashboard-%E2%86%92-T%3F?node-id=236-7516&node-type=frame&m=dev), the user needs to tap on "Save" to save the status. Would it be better to change the preference to `hasUserSavedDonorHistory`? @dbrant @Williamrai

Reply:
yes, we can do that.